content/next-docs/release-notes/release-notes-1.9.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release 1.9
+description: 'cert-manager release notes: cert-manager v1.9'
+---
+
+## v1.9.0
+
+cert-manager 1.9 adds alpha support for using cert-manager `Certificate`s in scenarios where the ordering of the RDN sequence that constitutes an X.509 certificate's subject needs to be preserved, improves ability to configure the `Certificate` created via ingress-shim using annotations on the `Ingress` resource and introduces various changes/improvements in contributor flow, most importantly defines a new make based contributor workflow.
+
+### Major Themes
+
+#### `LiteralCertificateSubject` feature
+
+cert-manager's `Certificate` allows to configure the subject fields of the X.509 certificate via `spec.subject` and `spec.commonName` fields. The [X.509 spec](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.6) states that the subject is an (ordered) sequence of Relative Distinguished Names (RDN). cert-manager does not strictly abide by the spec when encoding the subject fields from `Certificate`'s spec, for example, the order of the RDN sequence may not be preserved.
+For vast majority of the users this does not matter, but there are specific cases that require to be able to define the exact ordered RDN sequence, for example, if the certificate is used for LDAP authentication and the RDN sequence represents a [location in LDAP directory tree](https://ldapwiki.com/wiki/Directory%20Information%20Tree), see [`cert-manager#3203`](https://github.com/cert-manager/cert-manager/issues/3203).
+For these use cases, a new alpha `LiteralSubject` field has been added to `Certificate`'s spec where users can pass an RDN sequence, for example:
```

Comment:
There's one here in:
> `Certificate`'s spec

I understand there's technically a distinction, and would suggest this:

```suggestion
For these use cases, a new alpha `LiteralSubject` field has been added to the `Certificate` spec where users can pass an RDN sequence, for example:
```